src/common/modify-settings-modal/index.tsx

```diff
@@ -364,16 +384,41 @@ const ModifySettingsModal = ({ position, open, onCancel }: ModifySettingsModalPr
         color: 'primary',
         variant: 'contained',
         label: (
-          <FormattedMessage
-            description="Allow us to use your coin"
-            defaultMessage="Approve {token}"
-            values={{
-              token: fromToUse.symbol,
-            }}
-          />
+          <>
+            <FormattedMessage
+              description="Allow us to use your coin"
+              defaultMessage="Approve Max {token}"
+              values={{
+                token: fromToUse.symbol,
+              }}
+            />
+            <StyledTooltip
+              title={`You must give the Mean Finance smart contracts permission to use your ${fromToUse.symbol}`}
+              arrow
+              placement="top"
+            >
+              <HelpOutlineIcon fontSize="small" />
+            </StyledTooltip>
+          </>
         ),
-        onClick: handleApproveToken,
+        onClick: () => handleApproveToken(),
         disabled: !!hasPendingApproval,
+        options: [
+          {
+            text: (
+              <FormattedMessage
+                description="Allow us to use your coin"
```

Comment:
this description should be different since they are used as the uniq ids for the messages once we want tot ranslate them